loadtest/control/actions.go

```diff
@@ -471,6 +471,40 @@ func SearchUsers(u user.User) UserActionResponse {
 	})
 }
 
+// AutoCompleteUsers returns "user found" error when the auto completion is
+// done successfully. In that case UserActionResponse.Info carries the user
+// name information.
+func AutoCompleteUsers(u user.User) UserActionResponse {
```

Comment:
Does this need to be a `UserAction` or could it just be an helper function? I don't see this being called independently at the moment. Also using `Err` to signal "user found" doesn't look great.

Moreover, what if instead of passing the user interface we pass a function that runs the API call so we don't have to share the whole user object with a lower level function like this one.

Reply:
> Also using Err to signal "user found" doesn't look great.

:)